lib/sqlalchemy_ingres/base.py

```diff
@@ -157,12 +157,14 @@ def visit_sequence(self, seq, **kwargs):
     def limit_clause(self, select, **kwargs):
         # NOTE this now silently ignores keyword argument 'literal_binds', 'enclosing_alias', 'include_table', etc.
         text = ""
-        
         if not self.is_subquery():
-            if select._offset:
+            if select._offset is not None and select._offset > 0:
```

Comment:
seeing a pattern of checks here.

What do you think about a small utility function similar to:

    def is_integer_greater_than_zero(check_value):
        return check_value is not None and check_value > 0

and then using that instead?

Reply:
Any thoughts on this suggestion?

Reply:
Thanks for the suggestion! Utility function added to base.py for checking if value is greater than zero.